### PR TITLE
add arguments in deploy.sh for logging level

### PR DESCRIPTION
**Pull request by mikedoan**
add arguments in deploy.sh for logging level

### CODE REVIEW

**Comment by rlan35** [comment]
Jenkins, please test this.

**Comment by rlan35** [comment]
Please sync to master so the jenkins can pass.

**Comment by mikedoan** [comment]
Jenkins, test this please.

File: p2p/host/hostv2/hostv2.go [hunk 1]
```diff
@@ -112,16 +112,15 @@ func (host *HostV2) GroupReceiver(group p2p.GroupID) (
 
 // AddPeer add p2p.Peer into Peerstore
 func (host *HostV2) AddPeer(p *p2p.Peer) error {
+	if p.PeerID == "" {
+		host.logger.Error("AddPeer PeerID is EMPTY", "addrs", p.Addrs)
+		return fmt.Errorf("AddPeer error: peerID is empty")
```
Comment (LeoHChen) [inline p2p/host/hostv2/hostv2.go]: this is redundant.  If you return fmt.errorf, no need to add a log line here.

Comment (mikedoan) [reply p2p/host/hostv2/hostv2.go]: i'll remove that - it was someone else code and i just moved it here

**Comment by mikedoan** [comment]
please allow to check in this script only.

**Pull request merged**